frontend/app/_layout.tsx

```diff
@@ -27,6 +27,7 @@ const StackLayout = () => {
   }
 
   return (
+    // auth provider
```

Comment:
we are no longer using an auth provider, could remove this

Reply:
sorry forgot to remove that